src/titiler/extensions/tests/test_wms.py

```diff
@@ -0,0 +1,373 @@
+
+        response = client.get(
+            "/wms",
+            params={
+                "VERSION": "1.3.0",
+                "REQUEST": "GetMap",
+                "LAYERS": cog,
+                "BBOX": "373185.0,373185.0,8019284.949381611,639014.9492102272,8286015.0",
+                "CRS": "EPSG:32621",
+                "WIDTH": 334,
+                "HEIGHT": 333,
+                "FORMAT": "image/png",
+            },
+        )
+        assert response.status_code == 400
+        assert "Invalid 'BBOX' parameters" in response.text
+
+        # 1.3.0 needs inverted X,Y coordinates for EPSG:4326
+        response = client.get(
+            "/wms",
+            params={
+                "VERSION": "1.3.0",
+                "REQUEST": "GetMap",
+                "LAYERS": cog,
+                "BBOX": "72.22979795551834,-61.28762442711404,74.66298001264106,-52.301598718454485",
+                "CRS": "EPSG:4326",
+                "WIDTH": 334,
+                "HEIGHT": 333,
+                "FORMAT": "image/tiff; application=geotiff",
+            },
+        )
+        assert response.status_code == 200
+        meta = parse_img(response.content)
+        assert meta["crs"] == CRS.from_epsg(4326)
+        assert meta["bounds"] == [
+            -61.28762442711404,
+            72.22979795551834,
+            -52.301598718454485,
+            74.66298001264106,
+        ]
+
+        response = client.get(
+            "/wms",
+            params={
+                "VERSION": "1.1.1",
+                "REQUEST": "GetMap",
+                "LAYERS": cog,
+                "BBOX": "-61.28762442711404,72.22979795551834,-52.301598718454485,74.66298001264106",
+                "CRS": "EPSG:4326",
+                "WIDTH": 334,
+                "HEIGHT": 333,
+                "FORMAT": "image/tiff; application=geotiff",
+            },
+        )
+        assert response.status_code == 200
+        meta = parse_img(response.content)
+        assert meta["crs"] == CRS.from_epsg(4326)
+        assert meta["bounds"] == [
+            -61.28762442711404,
+            72.22979795551834,
+            -52.301598718454485,
+            74.66298001264106,
+        ]
+
+        response = client.get(
+            "/wms",
+            params={
+                "VERSION": "1.3.0",
+                "REQUEST": "GetMap",
+                "LAYERS": cog,
+                "BBOX": "-61.28762442711404,72.22979795551834,-52.301598718454485,74.66298001264106",
+                "CRS": "CRS:84",
+                "WIDTH": 334,
+                "HEIGHT": 333,
+                "FORMAT": "image/tiff; application=geotiff",
+            },
+        )
+        assert response.status_code == 200
+        meta = parse_img(response.content)
+        assert meta["crs"] == CRS.from_epsg(4326)
+        assert meta["bounds"] == [
+            -61.28762442711404,
+            72.22979795551834,
+            -52.301598718454485,
+            74.66298001264106,
+        ]
```

Comment:
Should we implement the mosaic method in this PR as well, or wait until a future version? I have a working prototype in my local rats nest, happy to wait or push to a later date :)

Reply:
because mosaic backend do not implement `part()` method, I'm not planing to add it